app/src/main/java/zapsolutions/zap/pin/PinFragment.java

```diff
@@ -286,6 +289,19 @@ public void onClick(View view) {
             }
         });
 
+        mBtnPinRemove.setOnClickListener(new OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                PrefsUtil.edit().putString(PrefsUtil.PIN_HASH, "").commit();
```

Comment:
Maybe a more descriptive way would be: `PrefsUtil.edit().remove(PrefsUtil.PIN_HASH).commit();`